src/wallet/wallet.h

```diff
@@ -712,8 +724,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * keystore implementation
      * Generate a new key
      */
-    CPubKey GenerateNewKey();
-    void DeriveNewChildKey(CKeyMetadata& metadata, CKey& secret);
+    CPubKey GenerateNewKey(bool internal = false);
+    void DeriveNewChildKey(CKeyMetadata& metadata, CKey& secret, bool internal = false);
```

Comment:
Do we want `internal` to have a default? Should it be `false`? IIRC currently new keys default to change unless added to the address book, so this default seems to contradict the status quo.